input/data/2011/2011A%26A...531L..18H/tev-000109.yaml

```diff
@@ -0,0 +1,25 @@
+source_id: 83
+reference_id: 2011A&A...531L..18H
+
+data:
+  livetime: 62
+  significance: 7.5
+
+pos:
+  ra: {val: 17h47m49s, err: 1.8m, err_sys: 1.3s}
+  dec: {val: -24d48m30s, err: 36s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 9.6, err: 2.4}
```

Comment:
`sigma` and `sigma2` have default units arcmin according to the schema?
If yes, then want you put is correct.

I would still suggest we change to be explicit with units in inputs, and long-term remove the default, i.e. only accept inputs with units specified. @pdeiml - Agree that it would be less error-prone and easier to review / maintain input data that way?

If yes, I'd suggest you put "m" here to be explicit, and then next week your or I make the change systematically.